starter-code/apple.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>Apple Pie Recipe</title>
+<link rel="stylesheet" href="css/main.css">
+<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500" rel="stylesheet">
+</head>
+<body>
+<header class="masthead" id="pie">
+  <div class="container">
+    <div class="intro-lead-in">
+      <h1>Apple Pies</h1>
+    </div>
+  </div>
+</header>
+<section id="text">
+  <div class="container">
+    <p class="big">This was my grandmother's apple pie recipe. I have never seen another one
+quite like it. It will always be my favorite and has won me several first place
+prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+	  <div class="text-center">
+	  <img src="images/apple-recipe-info.png" alt="time" class="image-c"/>
+	  </div>
+    <hr>
+    <div id="ingredients">
+      <h2>Ingredients</h2>
+      <ul class="recipe">
+        <li>1 recipe pastry for a 9 inch double crust pie</li>
+	<li>1/2 cup unsalted butter</li>
+	<li>3 tablespoons all-purpose flour</li>
+	<li>1/4 cup water</li>
+	<li>3 tablespoons all-purpose flour</li>
+	<li>1/4 cup water</li>
+	<li>1/2 cup white sugar</li>
+	<li>1/2 cup packed brown sugar</li>
+	<li>8 Granny Smith apples - peeled, cored and sliced</li>
+      </ul>
+    </div>
```

Comment:
Be consistent in your code formatting, this is very important! `<li>` should always be formatted inside the `<ul>`